debezium-connector-mysql/example-database.properties

```diff
@@ -0,0 +1,4 @@
+database.hostname=localhost
```

Comment:
Why is it we need this file? Can't we hard-code the default values, similar to `io.debezium.connector.postgresql.TestHelper.defaultJdbcConfig()`? The advantage being that now out-of-band information (separate properties file, launch arguments) is needed to run a test out of the box, e.g. when checking out the project and provided the DB server is running at the expected default location. The method providing the default configuration could still allow the values to be overridden by means of VM arguments of course.

To take a step, here are the goals I'd have in mind:

* Tests can be launched as-is in the IDE after checking out the project, without the need for any configuration (provided the DB is running, e.g. via `mvn docker:build docker:start -pl debezium-connector-mysql`)
* Using the same approach/code across different connectors to reduce friction